docs/charts.md

```diff
@@ -25,6 +25,7 @@ wordpress/
   Chart.yaml        # A YAML file containing information about the chart
   LICENSE           # OPTIONAL: A plain text file containing the license for the chart
   README.md         # OPTIONAL: A human-readable README file
+  NOTES.txt         # OPTIONAL: A plain text file containing short usage notes
```

Comment:
@onbjerg mostly to make it obvious it is a plain text file, we decided on .txt here https://github.com/kubernetes/helm/issues/993#issuecomment-238682762, but I'm open to other ideas if you have strong objections.